k2/csrc/cuda/debug_test.cu

```diff
@@ -56,7 +56,7 @@ __global__ void HelloCUDA(float f) {
 }
 
 // A vector add kernel definition
-__global__ void VecAdd(float* A, float* B, float* C) {
+__global__ void VecAdd(float *A, float *B, float *C) {
```

Comment:
I find that you have deleted the comment about adding `const` for the two input arguments `A` and `B`. 

The signature of this function tells us `A` and `B` will be changed inside the function,
but actually they are not. So it is really confusing for such a signature without `const`.

I would give a -1 vote.

Reply:
Agreed, should use const where possible.
Also please add the extra checking macros like K2_CHECK_GE and so on.

Reply:
I didn't delete things or IDK what you mean. The comment of issue or code, as to the issue it's still above.. As to the code, I post the page of doc:
![image](https://user-images.githubusercontent.com/705582/91666541-292d1780-eb30-11ea-9ecc-8209b7f2ad79.png)